ui/index.html

```diff
@@ -2,6 +2,7 @@
 
 <head>
   <title>DDNS Updater</title>
+  <link rel="icon" href="/favicon.ico" type="image/x-icon">

```

Comment:
would this work with an svg? If so that's awesome, I didn't know that!

Reply:
It would work but only on some browsers

```html
<link rel="icon" href="/favicon.svg" type="image/svg+xml">
```

see https://caniuse.com/link-icon-svg

Reply:
Sorry for the misunderstanding, I didn't see the favicon.ico file added.

Now I would suggest (nitpicking hard - sorry! 😄) that we either:

1. Add `<link rel="icon" href="/favicon.svg" type="image/svg+xml">` as well to index.html; or
2. Move the favicon.svg elsewhere so it's not part of the Docker image? / or add it to `.dockerignore` which might be easier/simpler?

Reply:
I excluded the file, imo it's a bit early to use a svg as favicon. To have both tags living together with one being the fallback of the other, there is a tag for that, but this fallback doesn't seem to be supported by android browsers `<link rel="alternate icon" href="/favicon.ico">`